src/core/main.c

```diff
@@ -1998,6 +1998,9 @@ static int invoke_main_loop(
                                 /* Reloading failed before the point of no return.
                                  * Let's continue running as if nothing happened. */
                                 m->objective = MANAGER_OK;
+                        else
+                                log_info("Reloading finished in " USEC_FMT " µs.",
```

Comment:
Hmm, on second thought. This might print something like "5273290 µs", which is not that great. The reload takes hundreds of ms or more, so the extra precision is just noise. Maybe round to ms?

Reply:
I know the bikeshed is strong in me today. Feel free to ignore this comment.

Reply:
truncated to ms

Reply:
```
[1237218.619141] image systemd[1]: Reloading...
[1237219.416869] image systemd[1]: Reloading finished in 796 ms.
```